integr8ly/installation
# Add pod count checks per namespace

To verify:

* Run a full install
* Go to the Prometheus Alerts page
* Verify the extra alerts are shown, and are all passing (with the exception of DeadMansSwitch) (see screenshot below)
* Scale down a pod in any of the monitored namespaces
* Verify the corresponding alert is active and yellow. It should turn Red if the pod stays down for 15 minutes
* Scale down *all* pods in any of the monitored namepsace
* Verify the corresponding alert is active
* Delete one of the monitored namespaces
* Verify the corresponding alert is active

![image](https://user-images.githubusercontent.com/878251/52440822-08990900-2b17-11e9-9027-81836a9289c8.png)
